lib/ansible/modules/cloud/vmware/vmware_guest.py

```diff
@@ -81,6 +81,12 @@
     - This is required if C(name) is not supplied.
     - If virtual machine does not exists, then this parameter is ignored.
     - Please note that a supplied UUID will be ignored on virtual machine creation, as VMware creates the UUID internally.
+  use_instance_uuid:
+    description:
+    - Use the VMWare instance UUID rather than the BIOS UUID.
```

Comment:
We usually start the description of boolean values with "Whether to use ...". Do this for all occurrences.
```suggestion
    - Whether to use the VMWare instance UUID rather than the BIOS UUID.
```